### ISSUE TITLE
Is the config tab broken?

### ISSUE THREAD

**Issue by BlueRaja** [aEnigmatic/xcom2-launcher#9]
Is the 'config' tab supposed to show .ini options available in each mod?  Because it is always empty for me.

**Comment by aEnigmatic** [comment]
Yes. I started to work on it, got frustrated with PropertyGrid and moved it to debug builds only. :see_no_evil:

**Comment by BlueRaja** [comment]
So use an ObjectListView with two columns, lol

**Comment by aEnigmatic** [comment]
Yeah, that was before discovering ObjectListView, haha.
